### PR TITLE
Add Introduction Statement to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bnb**
Adds a more friendly introduction to the CONTRIBUTING.md file. Feel free to comment on changes/updates!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Node.js Community Contributing Guide 1.0
+# Node.js Community Committee Contributing Guide 1.1
+
+**First things first:** If you're unsure of anything, please _don't hesitate ask a question in a discussion or create an issue_. Nobody will be upset with you; we'll always try to include you in the work you care about. The worst that can happen is that you'll be politely asked to change something. Any kind of contribution, no matter how big or small, helps !
```
Comment (vsemozhetbyt) [inline CONTRIBUTING.md]: Nits)
`don't hesitate ask` -> `don't to hesitate ask`
`helps !` -> `helps!`

**Comment by bnb** [comment]
@vsemozhetbyt fixed. Thanks for pointing those out 🙏

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Node.js Community Contributing Guide 1.0
+# Node.js Community Committee Contributing Guide 1.1
+
+**First things first:** If you're unsure of anything, please _don't hesitate to ask a question in a discussion or create an issue_. Nobody will be upset with you; we'll always try to include you in the work you care about. The worst that can happen is that you'll be politely asked to change something. Any kind of contribution, no matter how big or small, helps!
```
Comment (WaleedAshraf) [inline CONTRIBUTING.md]: 👍 LGTM
"The **worst** that can happen"
Not sure, if we can re-phrase this? 
There is nothing wrong but it just feels like something bad may happen.

Comment (chowdhurian) [reply CONTRIBUTING.md]: This comment got me thinking, but I'm undecided on making a suggestion. Thanks for raising!

Comment (bnb) [reply CONTRIBUTING.md]: I think I'm going to merge this for now, but TOTALLY open to a PR to change the phrasing! 💥

**Comment by bnb** [comment]
Merging 👍

**Pull request merged**